### PR TITLE
Remove disable minimize workaround on MacOS

### PR DESCRIPTION
**Pull request by haslinghuis**
Remove the workaround as it seems fixed with updated NWjs and Node versions.

Fixes: https://github.com/betaflight/betaflight-configurator/issues/2804

EDIT: Thanks @daleckystepan for testing. Seems the workaround is not needed anymore..

### CODE REVIEW

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Review by McGiverGim — DISMISSED** [review]

**Comment by daleckystepan** [comment]
Not good.
It works when I have window on my desktop and switch desktop.
BUT
It does not work if I have BF on fullscreen and switch desktop - it will restore itself on main desktop in smaller window.

**Review by daleckystepan — APPROVED** [review]

**Review by McGiverGim — APPROVED** [review]

**Review by Asizon — APPROVED** [review]

**Comment by robertlacroix** [comment]
@haslinghuis are you sure this workaround is not needed anymore? I just tested 4.4-RC4 and it seems the problem this workaround fixed is happening again.

It's easy to test: Press CMD+H to hide the Configurator window, and now there's no way to getting the window back. With the workaround it minimized instead so you re-open it from the dock.

I'd recommend putting the workaround back.

**Comment by robertlacroix** [comment]
The bug that @ctzsnooze filed with NWJS is also still unresolved: https://github.com/nwjs/nw.js/issues/7696

**Comment by haslinghuis** [comment]
Don't have Mac device to test on. Please provide info about OSX device and software version used.

> is happening again.

Did it work with previous release?

**Comment by robertlacroix** [comment]
Yes, it was working fine in 10.8, I guess ever since #2446 landed in a release.

**Comment by haslinghuis** [comment]
> Please provide info about OSX device and software version used.

**Comment by robertlacroix** [comment]
Problem reproduced on a machines running macOS 13.1 and 12.6.2. I understand the workaround causes #2804, but I think not having the workaround is worse, as you have to force quit Configurator after you press CMD+H (which many Mac users press frequently to get an app out of the way temporarily).

**Comment by haslinghuis** [comment]
@ctzsnooze please confirm

**Pull request merged**
